src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs

```diff
@@ -65,7 +65,7 @@ private void QueueObject(string v, WebhookMessageQueueObj obj, StorageType confi
 
 public interface IWebhookOperations
 {
-    Task SendEvent(EventMessage eventMessage);
+    Task SendEvent(ILogTracer log, EventMessage eventMessage);
```

Comment:
Where are we with the plan on having the LogTracer in the middleware ?
With the current solution we need to add that new parameter to every method
Can we bring back [this change](https://github.com/microsoft/onefuzz/pull/1765#discussion_r847519396) 
 and rely on the fact that a LogTracer will be generated for each request with a new ID